zeppelin-interpreter/pom.xml

```diff
@@ -213,6 +213,11 @@
       <exclusions>
       </exclusions>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+    </dependency>
```

Comment:
I think minimizing dependency of `zeppelin-interpreter` is important.
Because every interpreter implementation will have `zeppelin-interpreter` as a dependency, and more dependency in `zeppelin-interpreter` the higher chances of dependency conflict.

For example, spark interpreter may require different `guava` library version depends on SPARK_HOME on runtime.

I think making InterpreterOption independent from `guava` is not going to be too difficult.